arch/risc-v/src/mpfs/Kconfig

```diff
@@ -46,17 +46,12 @@ config MPFS_BOOTLOADER
 	---help---
 		This NuttX image is used as a bootloader, which will boot only on one hart, putting the others in WFI
 
-config IRQ_NSTACKS
```

Comment:
look like you remove both IRQ_NSTACKS and ARCH_CPU_COUNT, is it enough to only use SMP_NCPUS for your case?

Reply:
Yes, only SMP_NCPUS is needed.